src/backend/graphItemService.ts

```diff
@@ -218,7 +218,7 @@ export class GraphItemService {
       }
     }
     return {
-      propertyKeys: propertyKeys,
+      propertyKeys: parsedCSV.headers.slice(),
```

Comment:
why using slice here ?

Reply:
@renato-rg 
I have the same question as @aminerhanemi , why do we use `headers.slice()` and not just `headers` ?

Reply:
@amine @mzorkany just avoiding keeping a reference for parsedCSV.headers array in case propertyKeys or parsedCSV.headers are modified later